arbitrum-docs/node-running/partials/_local-dev-node-setup-instructions.mdx

```diff
@@ -50,65 +48,24 @@ docker exec nitro-testnode-sequencer-1 ls /config
 
 ## Token bridge
 
-<!-- This is a temporary block since the tokenbridge behavior is different in the "release" and stylus branches for now. -->
-<!-- Once the "stylus" branch is updated, we should remove this conditional and move the HTML format to MD -->
-
```

Comment:
Saw this comment, should we just change this to be a `.md` file now and remove the `CodeBlock` import?

Reply:
Yeah, we don't need to change the file extension, but just remove the HTML tags, and use plain Markdown in the file.